### ISSUE TITLE
Cannot find tool named 'proton_7'

### ISSUE THREAD

**Issue by jaq-the-cat** [muttleyxd/arma3-unix-launcher#203]
Log:
```[2022-02-27 13:55:01.082] [info] Started Arma 3 Unix Launcher, config file path: "/home/jaqi/.config/a3unixlauncher/launcher.conf"
[S_API] SteamAPI_Init(): Loaded '/home/jaqi/.local/share/Steam/linux64/steamclient.so' OK.
Setting breakpad minidump AppID = 107410
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561198207829768 [API loaded no]
[2022-02-27 13:55:11.406] [info] Steam path: /home/jaqi/.local/share/Steam
found: proton_7
[2022-02-27 13:55:11.407] [info] Install path: /home/jaqi/.local/share/Steam
[2022-02-27 13:55:11.407] [critical] Direct launch failed, exception: cannot find tool named 'proton_7'. Is it installed in Steam?```

**Comment by muttleyxd** [comment]
Can you run the launcher with `--verbose` parameter and post logs from it?

**Comment by jaq-the-cat** [comment]
I fixed it by downgrading again to Proton 6.3, the code apparently requires an update for the new version. I'll do that as soon as I can.

**Comment by muttleyxd** [comment]
Okay, I think I know what's the reason for this failure.

compatibility tools mapping VDF file specifies that it wants `proton_7` for Arma 3

A3UL doesn't have any kind of way to tell where this "proton_7" is installed, so it iterates over all apps installed by Steam and checks if any of them has a similar name. Problem is that Proton 7.0 gets converted to "proton_7" by the launcher and it doesn't match with `proton_7`. I'll fix the issue soon

**Comment by muttleyxd** [comment]
This should be fixed by #204 

Feel free to reopen if issue persists
